libs/common/src/helpers/date-validation.test.ts

```diff
@@ -203,6 +203,13 @@ describe('date-validation helpers', () => {
         year: '2050',
         expectedParsedDate: new Date('2050-12-12T00:00:00.0000'),
       },
+      {
+        description: 'Correct date with trailing white space',
```

Comment:
for completeness could you also add whitespace at the beginning (tbh could be one test with both)